### ISSUE TITLE
e2p1 file displays "address not found"

### ISSUE THREAD

**Issue by scoutperry** [susanBuck/e2-fall20#38]
Hello everyone,
So after Susan helped me with my Apache port issues I am now having some issues with pulling up my e2p1 file in the browser;

<img width="1053" alt="Screen Shot 2020-09-26 at 7 59 00 PM" src="https://user-images.githubusercontent.com/52388305/94352435-eaa55280-0032-11eb-81d3-b0ae07489a54.png">

http://localhost works and http://practice.loc/ works but http://e2p1 gives the IP address not found. My vhost setup is: 

```
<VirtualHost *:80>
    ServerName e2p1.loc
    DocumentRoot /Applications/XAMPP/xamppfiles/htdocs/e2/p1
    <Directory /Applications/XAMPP/xamppfiles/htdocs/e2/p1>
        AllowOverride All
        Options -Indexes
        Require all granted
    </Directory>
</VirtualHost>
```

So I tried to trouble shoot with the class notes and the feedback from my last forum submission. I tried the ping command Susan suggested

`$ ping e2p1.loc`

and got the response:

`ping: cannot resolve e2p1.loc: Unknown host`

I did a google search and found this stack overflow entry (https://bit.ly/3i6SvU3). I just wanted to see if other folks had any feedback before I try the stack overflow suggestions.

Thanks!

**Comment by tatthung** [comment]
Hi @scoutperry, 

Have you tried turning Apache on XAMPP? 

![image](https://user-images.githubusercontent.com/10286253/94355338-fab24600-00b5-11eb-8722-142e31818d9b.png)

**Comment by walshkelsey** [comment]
Have you tried other possible addresses? I ran into a similar issue trying to get my Digital Ocean site up. 

The way mine is set up for my local site it directs to e2p1.loc, though my digital ocean one is p1.kwalsh12.me. It took hours of troubleshooting to find out I didn't have e2 in front of the file name 🤦 .

**Comment by scoutperry** [comment]
hi @tatthung 
Yep, I have had apache is on for xampp. I think it might be something else.
Thanks anyway,
Scout

**Comment by scoutperry** [comment]
Hi @walshkelsey,
Well so it's my local server I'm trying to access, not my production server. I'm getting that display on my local server

**Comment by susanBuck** [comment]
Hi @scoutperry -

Based on the ping info you shared, it seems like the local domain e2p1.loc is not properly resolving to the local server IP address of 127.0.0.1.

Can you share the output of your computer's **hosts** file so we can double check that? (Refer to these notes for the location of this file: https://hesweb.dev/e2/notes/local-server/local-domains#step-3-create-a-new-host)

**Comment by scoutperry** [comment]
Ok I see. So the host file reads

`127.0.0.1 practice.loc`

should I be adding the same line with e2p1.loc or replacing it? I tried both and I'm still having the same issues

**Comment by susanBuck** [comment]
You want to add it - for every local domain you create, a new line gets added.

e.g. it should look like this:

```
127.0.0.1 practice.loc
127.0.0.1 e2p1.loc
```

Note how each one is on its own line.

Try that as I've written, then save your changes to the hosts file.

Then, run `cat /private/etc/hosts` to view the contents of your hosts file and confirm it looks as expected. Share a screenshot here just to be sure. : )

Finally, run `ping e2p1.loc` as you did early and see if it resolves to `127.0.0.1`.

**Comment by scoutperry** [comment]
Yes! That worked, thank you. And here's the screenshot

<img width="712" alt="Screen Shot 2020-09-28 at 9 11 03 AM" src="https://user-images.githubusercontent.com/52388305/94437139-5b4f8a80-016b-11eb-9c47-1b044e9aa5b3.png">

Closing the thread now!
